actions/workflows/test_range.yaml

```diff
@@ -112,7 +116,12 @@ tasks:
     next:
       - when: "{{ succeeded() and ctx().next_free_octet <= ctx().reserved_address_max_value }}"
         publish:
-          - err_message: "addresses are not claimed"
+          - err_message: "Error claiming an IP address in Men&Mice within the range {{ ctx().range_name }}. When trying to claim 
```

Comment:
Adding line breaks like this isn't going to work, check out: https://yaml-multiline.info/